### PR TITLE
Restructure docs html to match https://segment.com/docs/ in structure

### LINKED ISSUE #9: mvp4 - Create a top navigation pane, similar to https://segment.com/docs/
https://github.com/Shippable/pm/issues/8293

### LINKED ISSUE #10: mvp4 - Add the left navigation pane with up to 2 levels. Headings in the left pane can be expanded or collapsed. Clicking on any subheading opens an html page.
https://github.com/Shippable/pm/issues/8293

### PR DESCRIPTION
**Pull request by 0x-x0**
### UPDATED GIF

anim

closes https://github.com/Shippable/docs/issues/9
closes https://github.com/Shippable/docs/issues/10

### CODE REVIEW

File: theme/saas/static/css/shippable.css [hunk 1]
```diff
@@ -142,3 +142,284 @@ blockquote {
...
+
+/* ------------------ */
+.shipContent {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-align: stretch;
+    align-items: stretch;
+    -ms-flex-direction: row-reverse;
+    flex-direction: row-reverse;
+    height: calc(100vh - 63px)
```
Comment (rageshkrishna) [inline theme/saas/static/css/shippable.css]: What does this do?

File: theme/saas/static/css/shippable.css [hunk 2]
```diff
@@ -142,3 +142,284 @@ blockquote {
...
+    text-align: left;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-pack: center;
+    justify-content: center
+}
+
+.shipDocsNav-tab {
+    display: inline-block;
+    font-size: 16px;
```
Comment (rageshkrishna) [inline theme/saas/static/css/shippable.css]: I think font sizes should be in relative units. Using pixels will not allow things to scale uniformly if we change the base font sizes. We need to be able to maintain the ratios across the board if we want to change the sizes later.

File: theme/saas/static/css/shippable.css [hunk 3]
```diff
@@ -142,3 +142,284 @@ blockquote {
...
+    height: 16px;
+    background-image: url(../ionicons/css/search.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+    top: 13px;
+    left: 16px
+}
+
+.shipDocsNav-search-input {
+    font-size: 16px;
```
Comment (rageshkrishna) [inline theme/saas/static/css/shippable.css]: Same as above; px sizing for fonts is a bad idea.

File: theme/saas/static/css/shippable.css [hunk 4]
```diff
@@ -142,3 +142,284 @@ blockquote {
...
+    margin-bottom: .5rem
+}
+
+.shipNav-item.active {
+    z-index: 100
+}
+
+.shipNav-item-header {
+    color: #425e78;
+    font-size: 16px;
```
Comment (rageshkrishna) [inline theme/saas/static/css/shippable.css]: Same as above; I think font sizes should be relative.

File: theme/saas/static/css/shippable.css [hunk 5]
```diff
@@ -142,3 +142,284 @@ blockquote {
...
+    padding-bottom: 1rem
+}
+
+.shipContent-sidebar-title:empty {
+    display: none
+}
+
+.shipContent-main {
+    /*max-width: 800px;*/
+    padding: 3rem 4rem 8rem 96px
```
Comment (rageshkrishna) [inline theme/saas/static/css/shippable.css]: I don't understand why everything is relative except the last one.

**Review by rageshkrishna — CHANGES_REQUESTED** [review]

**Comment by rageshkrishna** [comment]
There are a lot of "magic numbers" in the CSS. Is there a way to do things in a cleaner way?

**Comment by manishas** [comment]
@chetantarale we spoke about the top nav yesterday but I still do not see it in this image.

**Comment by 0x-x0** [comment]
Updated  with latest image

**Comment by 0x-x0** [comment]
Merged this into a single PR to test the links. therefore closing this

**Pull request closed without merging**